docs/changelog.md

```diff
@@ -10,6 +10,18 @@ We use _breaking :warning:_ to mark changes that are not backward compatible \(r
 
 ## Unreleased
 
+## [v5.7.0](https://github.com/tellor-io/telliot/releases/tag/v5.7.0) - 2021.02.23
+
+### Changed
+ 
+* [\#403](https://github.com/tellor-io/telliot/pull/403) The config file now has fields separated by commands(eg. Dataserve, mine,etc) where all the command specific configuration goes. It now uses strict parsing, meaning that unused fields throw an error. See the `configs/config.json` for an example.
```

Comment:
We enabled sctrict parsing(fail when the config.json has non existing fields) so I imagine existing configs will fail to load and it would be good to mention this. Maybe again point to the example config in the configs/ folder or look at the actual default config struct for a better idea how the new config looks like. One of the next tasks is to auto generate the cli doc page so that we don'thave to explain this anymore and just point to the page